content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,364 @@
+kind: MutatingAdmissionPolicyBinding
+metadata:
+  name: "sidecar-binding-test.example.com"
+spec:
+  policyName: "sidecar-policy.example.com"
+```
+
+Next, the policy can be tested by creating a pod:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  initContainers:
+  - name: myapp-initializer
+    image: example/initializer:v1.0.0
+  containers:
+  - name: myapp
+    image: example/myapp:v1.0.0
+```
+
+Since the API request to create it matches the MutatingAdmissionPolicy the pod will be mutated resulting in:
+
+```yaml
+apiVersion: v1
+kind: Pod
+spec:
+  ...
+  initContainers:
+  - name: mesh-proxy
+    image: mesh/proxy:v1.0.0
+    args: ["proxy", "sidecar"]
+    restartPolicy: Always
+  - name: myapp-initializer
+    image: example/initializer:v1.0.0
+  containers:
+  - name: myapp
+    image: example/myapp:v1.0.0
+```
+
+#### Parameter Resources
+
+Parameter resources allow a policy configuration to be separate from its
+definition. A policy can define paramKind, which outlines GVK of the parameter
+resource, and then a policy binding ties a policy by name (via policyName) to a
+particular parameter resource via paramRef.
+
+If parameter configuration is needed, the following is an example of a
+MutatingAdmissionPolicy with parameter configuration.
+
+```yaml
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: MutatingAdmissionPolicy
+metadata:
+  name: "sidecar-policy.example.com"
+spec:
+  paramKind:
+    kind: Sidecar
+    apiVersion: mutations.example.com/v1
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["apps"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE"]
+      resources:   ["pods"]
+  matchConditions:
+    - name: does-not-already-have-sidecar
+      expression: "!object.spec.initContainers.exists(ic, ic.name == params.name)"
+  failurePolicy: Fail
+  reinvocationPolicy: IfNeeded
+  mutations:
+    - patchType: "ApplyConfiguration"
+      applyConfiguration:
+        expression: >
+          Object{
+            spec: Object.spec{
+              initContainers: [
+                Object.spec.initContainers{
+                  name: params.name,
+                  image: params.image,
+                  args: params.args,
+                  restartPolicy: params.restartPolicy
+                }
+              ]
+            }
+          }
+```
+
+To configure an mutating admission policy for use in a cluster, a binding and
+parameter resource are created. The following is an example of a
+MutatingAdmissionPolicyBinding that uses a cluster-wide param - the same param
+will be used to validate every resource request that matches the binding:
+
+```yaml
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: MutatingAdmissionPolicyBinding
+metadata:
+  name: "sidecar-binding-test.example.com"
+spec:
+  policyName: "sidecar-policy.example.com"
+  paramRef:
+    name: "meshproxy-test.example.com"
+    namespace: "default"
+```
+
+The parameter resource could be as following:
```

Comment:
```suggestion
The parameter resource could be the following object:
```

Reply:
This particular parameter is a Sidecar, and Sidecar's API looks like an object (has `.metadata` and `.spec`, has `.metadata.name`, etc).